### PR TITLE
docs updates

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +20,14 @@ Review the [Architecture concept](docs/architecture.md) document for more detail
...
 - Online deployment of new [model versions](docs/model_version_policy.md).
+- Configuration updates a in runtime
 - Support for AI accelerators including [Intel Movidius Myriad VPUs](https://docs.openvinotoolkit.org/latest/openvino_docs_IE_DG_supported_plugins_VPU.html), 
 [GPU](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_supported_plugins_CL_DNN.html) and [HDDL](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_supported_plugins_HDDL.html). 
 - The server can be enabled both on [Bare Metal Hosts](docs/host.md) or in
 [Docker containers](docs/docker_container.md).
-- [Kubernetes deployments](deploy). The server can be deployed in a Kubernetes cluster allowing the inference service to scale horizontally and ensure high availability.  
 - [Model reshaping](docs/shape_and_batch_size.md). The server supports reshaping models in runtime. 
 - [Directed Acyclic Graph Scheduler](docs/dag_scheduler.md) Connect multiple models to deploy complex processing solutions and reduce overhead of sending data back and forth.
+- [Custom nodes in DAG pipelines](docs/custom_node_development.md) Model inference can be represented by a custom node implementation attached as a C/C++ library.
```
Comment (atobiszei) [inline README.md:30]: This sentence seems off to me. I think the intent here is to give message that user can implement any
"inference-like" action as a custom node inside DAG and that's why we mention it here beside DAG documentation?

I would rewrite this at least like:
```suggestion
- [Custom nodes in DAG pipelines](docs/custom_node_development.md) Model inference can be implemented by a custom node C/C++ implementation loaded as an external library.
```

File: docs/dag_scheduler.md [hunk 2]
```diff
@@ -41,6 +41,21 @@ Custom nodes are not versioned, meaning one custom node library is bound to one
...
+During the pipeline execution, it is possible to split a request with mulitple batches into a set of branches with a single batch.
+That way a model configured with a batch size 1, can process requests with arbitrary batch size. Internally, OVMS demultiplexer will
+divide the data, process them in parallel and combine the results. 
+
+De-multiplication of the node output is enabled in the configuration file by adding `demultiply_count`. 
+It assumes the batches are combined on the first dimension which is dropped after splitting. For example:
+- a node returns output with shape `[8,1,3,224,224]`
+- demuliplexer created 8 requests with shape `[1,3,224,224]`
+- next model process in parallel 8 requests with output shape `[1,1001]` each.
+- results will be combined into a single output with shape `[8,1,1001]`
```
Comment (atobiszei) [inline docs/dag_scheduler.md:55]: This is too vague as it can suggest that the next model will do the gathering.

File: docs/dag_scheduler.md [hunk 3]
```diff
@@ -41,6 +41,21 @@ Custom nodes are not versioned, meaning one custom node library is bound to one
...
+## Demultiplexing data
+
+During the pipeline execution, it is possible to split a request with mulitple batches into a set of branches with a single batch.
+That way a model configured with a batch size 1, can process requests with arbitrary batch size. Internally, OVMS demultiplexer will
+divide the data, process them in parallel and combine the results. 
+
+De-multiplication of the node output is enabled in the configuration file by adding `demultiply_count`. 
+It assumes the batches are combined on the first dimension which is dropped after splitting. For example:
+- a node returns output with shape `[8,1,3,224,224]`
+- demuliplexer created 8 requests with shape `[1,3,224,224]`
```
Comment (atobiszei) [inline docs/dag_scheduler.md:53]: Keep one tense
```suggestion
- demuliplexer creates 8 requests with shape `[1,3,224,224]`
```

File: docs/dag_scheduler.md (hunk 2 above)
Comment (atobiszei) [inline docs/dag_scheduler.md:55]:
```suggestion
- results are combined into a single output with shape `[8,1,1001]`
```

File: README.md [hunk 4]
```diff
@@ -100,6 +107,7 @@ Follow a [contributor guide](docs/contributing.md) and a [developer guide](docs/
 
 * [Speed and Scale AI Inference Operations Across Multiple Architectures](https://techdecoded.intel.io/essentials/speed-and-scale-ai-inference-operations-across-multiple-architectures/?elq_cid=3646480_ts1607680426276&erpm_id=6470692_ts1607680426276) - webinar recording
 
+* [What is new in OpenVINO Model Server C++]()
```
Comment (mzegla) [inline README.md:110]: missing link

File: docs/dag_scheduler.md [hunk 5]
```diff
@@ -41,6 +41,21 @@ Custom nodes are not versioned, meaning one custom node library is bound to one
...
+- next model process in parallel 8 requests with output shape `[1,1001]` each.
```
Comment (mzegla) [inline docs/dag_scheduler.md:54]:
```suggestion
- next model processes in parallel 8 requests with output shape `[1,1001]` each.
```

File: docs/installations_kubernetes.md [hunk 6]
```diff
@@ -1,175 +1,5 @@
...
-GCS:
-```shell script
-$ gsutil rm -r gs://model-repository
-```
-
-S3:
-```shell script
-$ aws s3 rb s3://models-repository
-```
+There are two recommended methods or deployment of the OpenVINO Model Server in Kubernetes:
```
Comment (mzegla) [inline docs/installations_kubernetes.md:3]: Shouldn't be "methods **of** deployment"?

**Review by atobiszei — APPROVED** [review]

**Review by mzegla — APPROVED** [review]

**Pull request merged**